config/prometheus/prometheus-roles.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: prometheus
+rules:
+- apiGroups: [""]
+  resources:
+  - nodes
+  - nodes/proxy
+  - services
+  - endpoints
+  - pods
+  verbs: ["get", "list", "watch"]
+- apiGroups:
+  - extensions
+  resources:
+  - ingresses
```

Comment:
Same as above, do we need give prometheus the ingresses resources access? (not sure exactly what it means, but seems related to NGINX).

Reply:
Yup I'll look into this, thank you.